cabal.haskell-ci

```diff
@@ -3,5 +3,3 @@ unconstrained:          False
 hlint:                  True
 apt:                    freeglut3-dev
 irc-channels:           irc.freenode.org#haskell-lens
--- Work around https://github.com/haskell/haddock/issues/242
-haddock:                <7.6 || >=7.8
```

Comment:
Out of curiosity, how did you figure out that this was no longer necessary? I have to use this hack on other projects as well, and I'd be interested to know if there is a general approach to removing the hack.

Reply:
We dropped support for older GHCs :)

Reply:
Oho. Right you are. (We really should put this fact in the `CHANGELOG`!)